src/plugins/browser-console/index.ts

```diff
@@ -0,0 +1,21 @@
+import { BasePlugin } from '../../base';
+import type { PluginFactory } from '../../types';
+import { BrowserConsolePublisher } from './publisher';
+
+/**
+ * Browser Console plugin
+ */
+export class BrowserConsolePlugin extends BasePlugin<never, never> {
+  name = 'browserConsole';
+  publishers = [new BrowserConsolePublisher()];
+}
+
+/**
+ * Browser Console plugin factory function
+ *
+ * Use this function to instantiate the BrowserConsolePlugin when registering
+ * this plugin within Stratum.
+ */
+export const BrowserConsolePluginFactory: PluginFactory<BrowserConsolePlugin> = () => new BrowserConsolePlugin();
```

Comment:
Add browser-console to the `package.json` file so that it can be directly imported by consuming apps.